packages/app/README.md

```diff
@@ -197,6 +197,29 @@ parameters to send to the server. If you use one of H5Grove's default API
 implementations, then you'll need to use this prop to pass the `file` query
 parameter as shown above.
 
+#### `getExportURL?: URL | (() => Promise<URL | Blob>) | undefined`
+
+The `DataProvider#getExportURL` method is used by the toolbars to generate URLs
```

Comment:
```suggestion
The `DataProvider#getExportURL` method is used by the toolbars to generate URLs and controls
```

More generally, I think it would be nice to give an example as all this is a bit abstract and hard to figure out.

Reply:
I've added three examples.